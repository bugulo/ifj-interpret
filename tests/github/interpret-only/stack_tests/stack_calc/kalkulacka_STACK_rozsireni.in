+
20
6
-
20
8
*
5
6
/
15
3
e
