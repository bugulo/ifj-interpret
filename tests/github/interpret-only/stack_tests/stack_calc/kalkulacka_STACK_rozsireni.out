Zadejte vaši operaci (+,-,*,/) nebo e pro ukončení:Zadejte číslo 1:
Zadejte číslo 2:
Váš výsledek: 26
Zadejte vaši operaci (+,-,*,/) nebo e pro ukončení:Zadejte číslo 1:
Zadejte číslo 2:
Váš výsledek: 12
Zadejte vaši operaci (+,-,*,/) nebo e pro ukončení:Zadejte číslo 1:
Zadejte číslo 2:
Váš výsledek: 30
Zadejte vaši operaci (+,-,*,/) nebo e pro ukončení:Zadejte číslo 1:
Zadejte číslo 2:
Váš výsledek: 5
Zadejte vaši operaci (+,-,*,/) nebo e pro ukončení: